python/pyspark/ml/tests.py

```diff
@@ -1243,6 +1244,45 @@ def test_tweedie_distribution(self):
         self.assertTrue(np.isclose(model2.intercept, 0.6667, atol=1E-4))
 
 
+class FPGrowthTests(SparkSessionTestCase):
+    def setUp(self):
+        self.shuffle_partitions = self.spark.conf.get("spark.sql.shuffle.partitions")
+        self.spark.conf.set("spark.sql.shuffle.partitions", "1")
```

Comment:
Why 1 partition?  If it's for speed, then I wouldn't bother, unless we want to adjust it for all unit tests.  (I agree that setting it to 4 or so is often much faster than the default of 200.)  But let's do that in another PR.